src/etc/inc/certs.inc

```diff
@@ -312,7 +314,7 @@ function cert_import(& $cert, $crt_str, $key_str) {
 	return true;
 }
 
-function cert_create(& $cert, $caref, $keylen, $lifetime, $dn, $type = "user", $digest_alg = "sha256") {
+function cert_create(& $cert, $caref, $keylen, $lifetime, $dn, $type = "user", $digest_alg = "sha256", $eckey = "brainpoolP160r1", $keytype = "RSA") {
```

Comment:
Why is `brainpoolP160r1` the default? We should probably at least use `brainpoolP256r1` for a default, since [NIST recommends using a minimum of 256 bit curves](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-57pt1r4.pdf).